### PR TITLE
[KL-501] implement building backend image

### PR DESCRIPTION
**Pull request by LikoIlya**

### CODE REVIEW

File: .github/workflows/backend-image.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            type=ref,event=tag
+            type=ref,event=pr
+            type=sha
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3
+        with:
+          file: ./backend.Dockerfile
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```
Comment (lourenc) [inline .github/workflows/backend-image.yml:66]: Consider passing Github Actions caching provider to build action.
i.e.
```yaml
cache-from: type=gha
ache-to: type=gha,mode=max
```

More information at: https://docs.docker.com/build/cache/backends/gha/

**Review by lourenc — CHANGES_REQUESTED** [review]
Looks great! Please add caching to the docker build step and let's merge it

File: .github/workflows/backend-image.yml (hunk 1 above)
Comment (LikoIlya) [reply .github/workflows/backend-image.yml:66]: Added)

**Review by lourenc — APPROVED** [review]
👍

**Review by unixvb — APPROVED** [review]

**Pull request merged**